vcstools/general_utils.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import logging
+from astropy.time import Time
+from time import strptime, strftime
+from astropy.utils import iers
+
+logger = logging.getLogger(__name__)
+
+def sfreq(freqs):
+
```

Comment:
Do you know what this function is meant to do?

Reply:
It's used to do that weird channel reordering, I think it's only required by the correlator now

Reply:
I'm just going to assume that it's fine